Blockzilla/DomainCompletion.swift

```diff
@@ -7,8 +7,9 @@ import AutocompleteTextField
 
 class DomainCompletion: AutocompleteTextFieldCompletionSource {
     private lazy var topDomains: [String] = {
-        let filePath = Bundle.main.path(forResource: "topdomains", ofType: "txt")
-        return try! String(contentsOfFile: filePath!).components(separatedBy: "\n")
+        let localeLang = Bundle.main.preferredLocalizations[0]
+        var fileName = localeLang != "en" ? "topdomains-" + localeLang : "topdomains"
```

Comment:
I think those are two good suggestions @bkmunar . Knock them out and then let's get this merged.